lib/app/Auth/PhoenixTransactionBridge.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Phoenix\Auth;
+
+use DoSomething\Gateway\Contracts\TransactionBridgeContract; // we import the transaction bridge contract
+use League\OAuth2\Client\Token\AccessToken;
+
+class PhoenixTransactionBridge implements TransactionBridgeContract {
+
+  /**
+   * Get the value of the given HTTP header.
+   *
+   * @return string
+   */
+  public function getHeader($name) {
+      return drupal_get_http_header($name);
+  }
+
+  /**
+   * Write a log message.
+   *
+   * @return void
+   */
+  public function log($message, array $details) {
+      watchdog('Transaction', $details, [], WATCHDOG_INFO);
```

Comment:
does this work as expected? I think the [args](https://api.drupal.org/api/drupal/includes%21bootstrap.inc/function/watchdog/7.x) are reversed? 

also, it might be useful to provide more info in the message, I think the standard we've been using is `(module_name, thing that happened)`